core/process/aru.php

```diff
@@ -135,49 +135,92 @@
 	//
 	####################################
 
-	case 'spawnlist_update':
-		// Recent spawn
-		// ------------
-		
-		if ($config->system->mythic_recents) {
-			// get all mythic pokemon ids
-			$mythic_pokemons = array();
-			foreach ($pokemons->pokemon as $id => $pokemon) {
-				if ($pokemon->spawn_rate < 0.01) {
-					$mythic_pokemons[] = $id;
-				}
-			}
-
-			// get last mythic pokemon
-			$req		= "SELECT pokemon_id FROM pokemon
-					   WHERE pokemon_id IN (".implode(",", $mythic_pokemons).")
-					   ORDER BY disappear_time DESC LIMIT 0,1";
-		} else {
-			// get last pokemon
-			$req		= "SELECT pokemon_id FROM pokemon ORDER BY disappear_time DESC LIMIT 0,1";
-		}
-		$result 	= $mysqli->query($req);
-		$recents	= array();
-		$data 		= $result->fetch_object();
-		$pokeid 	= $data->pokemon_id;
-		
-		if ($_GET['last_id'] != $pokeid) {
-			$html = '
-		
-			<div class="col-md-1 col-xs-4 pokemon-single" data-pokeid="'.$pokeid.'" style="display:none;">
-						
-				<a href="pokemon/'.$pokeid.'"><img src="core/pokemons/'.$pokeid.'.png" alt="'.$pokemons->pokemon->$pokeid->name.'" class="img-responsive"></a>
-				<p class="pkmn-name"><a href="pokemon/'.$pokeid.'">'.$pokemons->pokemon->$pokeid->name.'</a></p>
-			
-			</div>	
-				
-			';
-			
-			
-			echo $html;
-		}
-	
-		break;
+    case 'spawnlist_update':
+        
+        // Recent spawn
+        // ------------
+        
+        if ($config->system->mythic_recents) {
+            // get all mythic pokemon ids
+            $mythic_pokemons = array();
+            foreach($pokemons->pokemon as $id=>$pokemon) {
+                if ($pokemon->spawn_rate < 0.01) {
+                    $mythic_pokemons[] = $id;
+                }
+            }
+            
+            // get last mythic pokemon
+            $req		= "SELECT pokemon_id, disappear_time, latitude, longitude, individual_attack, individual_defense, individual_stamina FROM pokemon
+                        WHERE pokemon_id IN (".implode(",", $mythic_pokemons).")
+                        ORDER BY disappear_time DESC LIMIT 0,1";
+        } else {
+            // get last pokemon
+            $req		= "SELECT pokemon_id, disappear_time, latitude, longitude, individual_attack, individual_defense, individual_stamina FROM pokemon ORDER BY disappear_time DESC LIMIT 0,1";
+        }
+        $result 	= $mysqli->query($req);
+        $data 		= $result->fetch_object();
+        $id 	= $data->pokemon_id;
+        $last_seen = strtotime($data->disappear_time)+60*60;    // TO-DO: implement soft coded time->delay
```

Comment:
I don't really get it, because we should first receive the disappear_time from the DB and store it, then change it to readable output with time_ago(*,*). The thing that needs to be addressed is the 3600s though.